bower.json

```diff
@@ -42,6 +42,7 @@
     "manageiq-ui-components": "~0.0.9",
     "moment-strftime": "~0.2.0",
     "moment-timezone": "~0.4.1",
+    "moment-duration-format": "~1.3.0",
```

Comment:
For now, you'll have to do the same change in `manageiq/bower.json`, we're in the process of moving it to ui-classic .. but .. right now, `manageiq/bower.json` is the one that ends up being used.